app.rb

```diff
@@ -1,9 +1,40 @@
+# frozen_string_literal: true
+
 require 'sinatra/base'
+require 'sinatra/reloader'
+require './lib/message'
+
+# class to pass params from forms to SQL
+class MessageManager < Sinatra::Base
+  configure :development do
+    register Sinatra::Reloader
+  end
+
+  get '/' do
+    erb :post_message
+  end
+
+  post '/messages' do
```

Comment:
You only need to use instance vars here if you want those vars to be available in the view.

Reply:
Nothing is broken by you using instance vars for things that are not needed in the view, but it's potentially confusing to the reader (and that's never good) :)